_data/payments.yml

```diff
@@ -105,6 +105,12 @@ websites:
       software: Yes
       doc: https://privacy.com/faq
 
+    - name: Ravelin
+      url: https://www.ravelin.com/
+      img: Ravelin.png
```

Comment:
if this is something we want to enforce, we should add it to travis and the help/template file